lib/nimble_template/addons/variants/phoenix/mix_release.ex

```diff
@@ -23,4 +25,61 @@ defmodule NimbleTemplate.Addons.Phoenix.MixRelease do
 
     project
   end
+
+  defp edit_files(%{otp_app: otp_app, web_module: web_module} = project) do
+    Generator.delete_content(
+      "config/runtime.exs",
+      """
+      # Start the phoenix server if environment is set and running in a release
+      if System.get_env("PHX_SERVER") && System.get_env("RELEASE_NAME") do
+        config :#{otp_app}, #{web_module}.Endpoint, server: true
+      end
+
+      """
+    )
+
+    Generator.delete_content(
+      "config/runtime.exs",
+      """
+
+        # ## Using releases
+        #
+        # If you are doing OTP releases, you need to instruct Phoenix
+        # to start each relevant endpoint:
```

Comment:
I think we can rephase this sentence to remove `you` as it isn't appropriate in technical comments 
```suggestion
        # For OTP releases, instruct Phoenix to start each relevant endpoint:
```

Reply:
same above, it comes from the default Phoenix @longnd :D we just remove it